develop.sh

```diff
@@ -5,5 +5,86 @@ if [[ ! -d "docker" ]]; then
     exit -1
 fi
 
-# invoke docker, but ungrep noisy influx log messages that I can't turn off
-docker-compose -f docker/docker-compose.yml -p listenbrainz build && docker-compose -f docker/docker-compose.yml -p listenbrainz up
+# Warn user if no arguments passed to develop.sh
+if [ "$#" == 0 ]; then
+	echo "No arguments provided"
```

Comment:
If we don't do a check here, docker will prompt a verbose message telling the user about *usage*, *options* and *commands*

Reply:
But yes, I understand what you say :)

Reply:
I don't think that it's a problem if we show this usage message. I think that we shouldn't be hiding the underlying tools from developers. In this case, we should have clear instructions in the readme telling people what to do, and also explain about the underlying tools so that they can use them if they need to. As I said during the summit, I think it makes sense for this script to help people do the repetitive tasks (add the -p and -f flags to docker-compose, correctly add --rm when needed, construct a psql commandline), but keep out of their way if they need to do something slightly different with docker-compose